src/systemc/tlm_bridge/sc_peq.hh

```diff
@@ -37,8 +37,8 @@
 #ifndef PAYLOAD_EVENT_H_
 #define PAYLOAD_EVENT_H_
 
-// TLM includes
-#include <tlm.h>
+// header file required by sc_core::sc_time
+#include "../ext/core/sc_main.hh"
```

Comment:
I only see '../' in `#include` lines in the src/systemc/ext/ directory, not outside of it. I think you should use `#include "systemc/ext/core/sc_main.hh"` instead.